src/raster/mdarray.rs

```diff
@@ -440,13 +451,45 @@ impl<'a> Group<'a> {
             Ok(Attribute::from_c_attribute(c_attribute))
         }
     }
+
+    pub fn dimensions(&self, options: CslStringList) -> Result<Vec<Dimension>> {
+        unsafe {
+            let mut num_dimensions: usize = 0;
+            let c_dimensions = GDALGroupGetDimensions(
+                self.c_group,
+                std::ptr::addr_of_mut!(num_dimensions),
```

Comment:
I'm not necessarily against it, but I'd expect to see `addr_of_mut!` used with uninitialized variables, so it's a bit confusing here.

Reply:
[ccfa51d](https://github.com/georust/gdal/pull/291/commits/ccfa51d913579b45473c530b051b1249bf807a32)